tests-functional/tests/test_wakuext_message_reactions.py

```diff
@@ -106,3 +107,71 @@ def test_one_to_one_message_reactions(self, backend_new_profile, waku_light_clie
                     item["emoji"] == "🙂",
                 )
             )
+
+    @pytest.mark.parametrize("waku_light_client", [False], indirect=True, ids=["wakuV2LightClient_False"])
+    def test_limit_of_20_reactions(self, backend_new_profile, waku_light_client):
+        """Test that you cannot send more than 20 message reactions on a single message"""
+        # Initialize two backends (sender and receiver) for this test
+        self.sender = backend_new_profile("sender", waku_light_client=waku_light_client)
+        self.receiver = backend_new_profile("receiver", waku_light_client=waku_light_client)
+
+        self.make_contacts(self.sender, self.receiver)
+        response = self.sender.wakuext_service.send_one_to_one_message(self.receiver.public_key, "test_message")
+        message = self.get_message_by_content_type(response, content_type=MessageContentType.TEXT_PLAIN.value)[0]
+        message_id, sender_chat_id = message["id"], message["chatId"]
+        receiver_chat_id = self.receiver.wakuext_service.rpc_request(method="chats")[0]["id"]
+
+        # Send 20 emojis
+        emojis = ["😀", "🤓", "😄", "😁", "😆", "😅", "🤣", "😂", "🥹", "🙂", "🙃", "😉", "😊", "😇", "🥰", "😍", "🤩", "😘", "😗", "☺️"]
```

Comment:
Would be nice to use `fake.emoji()` instead of hard-coding. You can check `utils/fake.py`.

Reply:
Actually, I just realized it's safer to hardcode them, because if we get twice the same random emoji, it will bug the test (like we have 19 emojis then the next line will not throw, thus failing the test